### PR TITLE
webgpu: upgrade webgpu/types

### PR DESCRIPTION
**Pull request by qjia7**
This PR also uses the platform supported max buffer size, otherwise the limit is 256M.
Meanwhile, it removes expired which is not supported anymore.

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

- - -
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7528)

### CODE REVIEW

File: tfjs-backend-webgpu/src/kernels/FromPixels.ts [hunk 1]
```diff
@@ -95,9 +86,8 @@ export function fromPixels(args: {
         if (fromPixels2DContext == null ||
             newWillReadFrequently !== willReadFrequently) {
           willReadFrequently = newWillReadFrequently;
-          fromPixels2DContext =
-              document.createElement('canvas').getContext(
-                  '2d', {willReadFrequently});
+          fromPixels2DContext = document.createElement('canvas').getContext(
```
Comment (shaoboyan) [inline tfjs-backend-webgpu/src/kernels/FromPixels.ts:89]: Just a format, right?

**Review by shaoboyan — APPROVED** [review]
LGTM for the `importExternalTexture`related changes.

File: tfjs-backend-webgpu/src/kernels/FromPixels.ts (hunk 1 above)
Comment (qjia7) [reply tfjs-backend-webgpu/src/kernels/FromPixels.ts:89]: Yes, it's automatically adjusted.

File: tfjs-backend-webgpu/src/kernels/FromPixels.ts [hunk 2]
```diff
@@ -72,21 +71,13 @@ export function fromPixels(args: {
...
-      const videoElement = pixels as HTMLVideoElement;
-      if (!(videoToTextureMap.has(videoElement)) ||
-          (videoToTextureMap.get(videoElement) as GPUExternalTexture).expired) {
-        const externalTextureDescriptor = {source: videoElement};
-        videoToTextureMap.set(
-            videoElement,
-            backend.device.importExternalTexture(externalTextureDescriptor));
-      }
-
+      const externalTextureDescriptor = {source: pixels as HTMLVideoElement};
```
Comment (xhcao) [inline tfjs-backend-webgpu/src/kernels/FromPixels.ts:74]: Nit: Remove `externalTextureDescriptor`, use `backend.device.importExternalTexture({source: pixels as HTMLVideoElement})`?

**Review by xhcao — APPROVED** [review]

**Review by gyagp — APPROVED** [review]
LGTM

**Pull request merged**